llvm/lib/Target/RISCV/RISCVInstrInfo.cpp

```diff
@@ -66,6 +66,22 @@ using namespace RISCV;
 
 } // namespace llvm::RISCVVPseudosTable
 
+namespace llvm::RISCVVInversePseudosTable {
```

Comment:
This should be put in `RISCVMCTargetDesc.cpp`

Reply:
MC layer doesn't need to know about pseudos. That just bloats llvm-mc and llvm-objdump code size for no reason.